### PR TITLE
examples: add kafka transform for multiple topics

### PR DESCRIPTION
**Pull request by srenatus**
With tests and a  try-at-home setup using docker-compose, redpanda, and benthos.

### CODE REVIEW

File: examples/kafka-multiple-topics/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Please see [tranform.rego](./transform.rego) for details.
+
+## Testing
+
+Use `eopa test .` (or `opa test .`) for running [the included unit tests](./transform_test.rego) against the transform policy.
+
+## E2E Testing
+
+In this directory, you find:
```
Comment (philipaconrad) [inline examples/kafka-multiple-topics/README.md:13]:
```suggestion
In this directory, you should find:
```
[typo]: Missing word?

File: examples/kafka-multiple-topics/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+-   A `docker-compose.yaml` that lets you spin up a local RedPanda instance with a handy web console on http://127.0.0.1:8080.
+-   A `benthos-kafka.yml` config that lets you use [Benthos](https://benthos.dev) to generate Kafka messages the three topics.
+-   An `enterprise-opa.yml` config that lets you spin up an Enterprise OPA instances configured to consume the tree topics.
```
Comment (philipaconrad) [inline examples/kafka-multiple-topics/README.md:17]:
```suggestion
-   An `enterprise-opa.yml` config that lets you spin up an Enterprise OPA instances configured to consume the three topics.
```
[typo]

**Review by philipaconrad — APPROVED** [review]
LGTM, just a few small typos. :slightly_smiling_face:

File: examples/kafka-multiple-topics/README.md (hunk 2 above)
Comment (charlesdaniels) [inline examples/kafka-multiple-topics/README.md:17]: s/tree/three/g ?

Comment (charlesdaniels) [reply examples/kafka-multiple-topics/README.md:17]: Philip already brought this up [here](https://github.com/StyraInc/enterprise-opa/pull/70#discussion_r1537791183).

**Comment by charlesdaniels** [comment]
```plain
$ docker-compose --version
docker-compose version 1.29.2, build unknown
$ docker-compose up
ERROR: The Compose file './docker-compose.yaml' is invalid because:
'name' does not match any of the regexes: '^x-'

You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
```

It could be that my `docker-compose` is just too old, but I haven't had this issue before.

**Comment by srenatus** [comment]
@charlesdaniels I've been using `docker compose`, not `docker-compose`, not sure but it could make a difference 🤔

**Comment by charlesdaniels** [comment]
Ahh, that does seem to have fixed it. Interesting. I was not aware those were different.

File: examples/kafka-multiple-topics/README.md (hunk 1 above)
Comment (srenatus) [reply examples/kafka-multiple-topics/README.md:13]: 🤷 You think so? I'd say _you find it_, because I know it's there 😄

**Pull request merged**